package in.co.rays.oop.ExceptionHandling;

public class Exceptionexample {

	public static void main(String[] args) {
		try { int a=5;
		 int b=0;
		 double div=a/b;
			
		} catch (Exception e) {
			 
			 System.err.println(e);
			
		}
		
	
	}
}
